public/app/themes/justice/inc/core.php

```diff
@@ -59,4 +62,42 @@ public function removeDefaultDashboardWidgets(): void
         remove_meta_box('dashboard_quick_press', 'dashboard', 'side');      // Quick Press/Quick Draft
         remove_meta_box('dashboard_activity', 'dashboard', 'normal');       // Activity
     }
+
+    /**
+     * Handle loopback requests.
+     *
+     * Handle requests to the application host, by sending them to the loopback url.
+     *
+     * @param false|array|WP_Error $preempt
+     * @param array $args
+     * @param string $url
+     * @return false|array|WP_Error
+     */
+
+    public function handleLoopbackRequests(false|array|WP_Error $preempt, array $args, string $url): false|array|WP_Error
```

Comment:
I found this slightly confusing buddy and had to look up the hook `pre_http_request`. 
The input variables are:

`$response`, `$parsed_args` and `$url`

Could we use the same names or indeed, make a note in the comment to map the meaning of the variable names?

Reply:
Done!